campaigns/workflows.md

```diff
@@ -0,0 +1,360 @@
+# The Workflows Campaign
+
+Improving Scientific Data Analysis Workflows
+
+## Overview
+
+In a very general sense, scientific data analysis workflows usually have a
+particular process structure to them:
+
+1. **Search & Discovery**: Locate the *starting point* for your analysis
+2. **Ingestion**: Open and read this data or code into a useful format
+3. **Analysis**: Perform some computations on the data
+4. **Check-point**: Save computed artifact as a new dataset
+5. **Publication**: Produce images and/or papers from the computed artifacts
+
+Implied by this process is the existence of a **Platform** for the user upon
+which each step in the above process is conducted.  *It is not necessary that
+the same platform be used for each step.*  In fact, multiple platforms *could*
+be used in a single step.  For example, during the *Search & Discovery* phase,
+the scientist might use a combination of web searching to find the larger
+dataset or collection, and then move to a Jupyter Notebook to make queries of
+the larger dataset to produce subsets.  It is not critical that a *single
+platform* be emphasized or prioritized, but making it easier and more
+productive for the scientist is critical.  That could mean reducing the number
+of platforms the scientist must use, but it could also mean making multiple
+platforms inter-operate better.
+
+It is extremely important to understand that this process, as described above,
+is just conceptual.  No scientist simply moves through this process *in order*
+nor in a *single pass*.  That is, scientists that perform actual analysis many
+times jump forward and back in the process.  For example, a scientist may jump
+forward from the *Ingestion* phase to the *Check-point* phase before doing any
+actual *Analysis*.  Or, a scientist may jump back to the *Search & Discovery*
+phase to find another dataset to compare with one they originally started
+analyzing.  A scientist might also *iterate* on their analysis, effectively
+making this process *cyclic*.  For example, a scientist might *publish* their
+Notebook with the expectation that their Notebook can be found during the
+*Search & Discovery* phase.
+
+## Vision
+
+The [Xdev Vision Statements](../MISSION.md#xdev-vision-statements) are easy
+to state in brief, but harder to state in detail.  We will try to describe what I
+see as the vision of the future of scientific workflows, based on the Pangeo
```

Comment:
> We will try to describe what I ...

A missed we -> I, but maybe something like

> Below we describe our vision of the future of scientific workflows, based on the Pangeo ecosystem.

Reply:
Good eye!  Fixed.